### PR TITLE
Update README with build instructions

### PR DESCRIPTION
**Pull request by yiannist**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,39 @@ It is entirely written in [Go](https://golang.org/) and backed by [Redis](https:
...
+
+Clone the repository or `go get` it with:
+```shell
+$ go get github.com/skroutz/downloader
+```
+
+Install `librdkafka-dev` in order to be able to build the project. For Debian
+GNU/Linux:
+```shell
+$ sudo apt install librdkafka-dev
```
Comment (eavgerinos) [inline README.md]: This is not yet a dependency. I think we should update readme after the new backend is available, or at least skip this part for now.

File: README.md [hunk 2]
```diff
@@ -5,6 +5,39 @@ It is entirely written in [Go](https://golang.org/) and backed by [Redis](https:
...
+Make sure that you have a working [Go
+environment](https://golang.org/doc/install) and that you have configured your
+[`$GOPATH`](https://golang.org/doc/code.html#Workspaces) correctly.
+
+Clone the repository or `go get` it with:
+```shell
+$ go get github.com/skroutz/downloader
+```
+
+Install `librdkafka-dev` in order to be able to build the project. For Debian
```
Comment (0xdiba) [inline README.md]: kafka is not a dependency in the `master` branch :) _yet_
Let's add it later...

File: README.md (hunk 1 above)
Comment (yiannist) [reply README.md]: Oh, right! I'll update the diff for now.

File: README.md [hunk 3]
```diff
@@ -120,20 +147,25 @@ Displayed Info:
 Development
 -------------------------------------------------------------------------------
 
-A Redis instance has to be installed for the downloader's tests to run.
+  You should have a *running* Redis instance in order to be able to run the
```
Comment (oliakremmyda) [inline README.md]: you could also use `:warning:` for  :warning: to give extra attention to this

**Pull request merged**